Dockerfile.aro-multistage

```diff
@@ -4,7 +4,7 @@
 # Currently the docker version on our RHEL7 VMSS uses a version which
 # does not support multi-stage builds.  This is a temporary stop-gap
 # until we get podman working without issue
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.7 AS builder
+FROM arointsvc.azurecr.io/ubi7/go-toolset:1.16.12 AS builder
```

Comment:
why are we downgrading the go-toolset version?

Reply:
Good catch! This has been fixed